README.md

```diff
@@ -92,6 +92,7 @@ Available variables are listed below, along with default values (see
 mariadb_use_official_repo: false
 mariadb_use_official_repo_url: https://deb.mariadb.org
 mariadb_use_official_repo_version: "10.10"
+mariadb_use_official_repo_deb822: false
```

Comment:
Can you use something more clear (like `mariadb_source_list_format_deb822`) and this is not related to using MDBF official vs debian repositories.

Reply:
We deploy a repository source only when using the 'use_official_repo', so it seem appropriate to keep the same prefixes.

Maybe ?
```
mariadb_use_official_repo_format_deb822
```

Reply:
yep true, ok